techmatters/flex-plugins
# CHI-1976 case timeline rework

## Description

- Limits the 'Timeline' section on the case details view to 5 items. If the timeline has more than 5 items, the heading changes to 'Recent Timeline' and a 'Show All Timeline Items' button is added below
- The 'See All Timeline Items' button opens a modal that displays a paginated view of all timeline items
- enable_separate_timeline_view added to enable above change
- Add capability to pass in substitution values to NavigableContainer for the title

### Checklist

### Related Issues
Fixes CHI-1976

### Verification steps

- ensure enable_separate_timeline_view is set 'true' for the test environment
- See tickets
